_docs/_latest/configure/scanner/default-rules.md

```diff
@@ -47,3 +47,6 @@ Forseti Scanner has default rules that create a
     * For major version 1.9, the minor version must be at least 7-gke.1
     * For major version 1.10, the minor version must be at least 2-gke.1
     * For major version 1.11, any minor version is allowed
+
+## Service Account Key
+  * User-managed service account keys should not be older than max age.
```

Comment:
What's the max age?

Reply:
Updated, to just say more generically as `certain time`.